drivers/telescope/lx200_10micron.cpp

```diff
@@ -771,7 +754,7 @@ bool LX200_10MICRON::SetTLEfromDatabase(int tleN)
     return 0;
 }
 
-bool LX200_10MICRON::CalculateTrajectory(int year, int month, int day, int hour, int minute, int nextpass, ln_date date_pass)
+bool LX200_10MICRON::CalculateSatTrajectory(char *start_pass_isodatetime, char *end_pass_isodatetime)
```

Comment:
const char * or better const std::string & if you can

Reply:
Done